### PR TITLE
Improved error logs when missing 'using' configuration in metadata file

### PR DESCRIPTION
**Pull request by AvaStancu**
Fixes [issue #2038](https://github.com/actions/runner/issues/2038)

### CODE REVIEW

**Comment by AvaStancu** [comment]
@TingluoHuang I understand you could confirm if 'plugin' is still supported besides 'using'. I've searched through the [metadata](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions) documentation but did not find anything for 'plugin' 🤷‍♀️

**Comment by TingluoHuang** [comment]
it's fine to leave `plugin` out here. 😄

File: src/Test/L0/Worker/ActionManifestManagerL0.cs [hunk 1]
```diff
@@ -698,6 +698,31 @@ public void Load_ConditionalCompositeAction()
...
+                Setup();
+
+                var actionManifest = new ActionManifestManager();
+                actionManifest.Initialize(_hc);
+                var action_path = Path.Combine(TestUtil.GetTestDataPath(), "composite_action_without_using_token.yml");
+
+                //Assert
+                var err = Assert.Throws<ArgumentException>(() => actionManifest.Load(_ec.Object, action_path));
+                Assert.Contains($"Fail to load {action_path}", err.Message);
+                _ec.Verify(x => x.AddIssue(It.Is<Issue>(s => s.Message.Contains("missing 'using' value. 'using' requires 'composite', 'docker', 'node12' or 'node16'.")), It.IsAny<string>()), Times.Once);
```
Comment (fhammerl) [inline src/Test/L0/Worker/ActionManifestManagerL0.cs:718]:
```suggestion
                _ec.Verify(x => x.AddIssue(It.Is<Issue>(s => s.Message.Contains("Missing 'using' value. 'using' requires 'composite', 'docker', 'node12' or 'node16'.")), It.IsAny<string>()), Times.Once);
```

Comment (fhammerl) [inline src/Test/L0/Worker/ActionManifestManagerL0.cs:718]:
```suggestion
                _ec.Verify(x => x.AddIssue(It.Is<Issue>(s => s.Message.Contains("Missing 'using' value. 'using' requires 'composite', 'docker', 'node12' or 'node16'.")), It.IsAny<string>()), Times.Once);
```

File: src/Runner.Worker/ActionManifestManager.cs [hunk 2]
```diff
@@ -503,7 +503,7 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
                 };
             }
 
-            throw new NotSupportedException(nameof(ConvertRuns));
+            throw new NotSupportedException("missing 'using' value. 'using' requires 'composite', 'docker', 'node12' or 'node16'.");
```
[the remark this answers is not in the document]
Comment (AvaStancu) [reply src/Runner.Worker/ActionManifestManager.cs]: Ammended, thanks!

File: src/Test/L0/Worker/ActionManifestManagerL0.cs (hunk 1 above)
Comment (AvaStancu) [reply src/Test/L0/Worker/ActionManifestManagerL0.cs]: Ammended, thanks!

Comment (AvaStancu) [reply src/Test/L0/Worker/ActionManifestManagerL0.cs]: This seems to be a duplicate

**Review by fhammerl — APPROVED** [review]
LGTM!

**Pull request merged**